.github/workflows/wheels-Windows-MacOS.yml

```diff
@@ -0,0 +1,55 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Wheels for Windows and MacOS
+
+on:
+  push:
+    branches:
+      - master
+  release:
+    types:
+      - published
+jobs:
+  build:
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [windows-2016, macOS-latest]
```

Comment:
`windows-2016` has been [planned to be removed a few times](https://github.blog/changelog/2019-10-31-github-actions-windows-server-2016-r2-virtual-environment-will-be-removed-on-november-7/) and is still here for the time being (https://github.com/actions/virtual-environments/issues/68#issuecomment-602652021, https://github.com/actions/virtual-environments) but will be going away at some point.

Is it possible to use `windows-2019` instead?

Reply:
The 2019 wouldn't compile. It looked like a kink in VS. I've lost the build traceback but the error was:
> 1>LINK : fatal error LNK1158: cannot run 'rc.exe'

I've googled around for that error and there does seem to be other fixes. I can dig deeper if you want?

Reply:
It would be good to use the `2019` as the `2016` will be going away sooner rather than later. But I'm happy enough to include a working `2016` right now and follow up later.

Reply:
I'll work on this as a second PR.